common/helpers.py

```diff
@@ -14,13 +14,10 @@ def tables(database, *table_names):
             ...
       Where vcfs and genotypes are tables in the provided db.
     """
-    try:
-        connection = database.engine.connect()
+    with database.engine.connect() as connection:
```

Comment:
Hmm, I wrote this code, but the `finally` (even if the connection were in scope) looks like it would be a problem? If the connection is closed, then it can't be used in the function using `tables`, right? Does this still work?